docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,83 @@
+
+Analyze disk usage
+=======================================================
+From a compute node, ``srun --pty /bin/bash``, run the following command from your ``/home/<username>`` directory: ::
+    
+ du -shc .[^.]* *
+
+This command will output the size of each file, directory, and hidden directory in your ``/home/<username>`` space, with the total of your ``/home`` directory being the last line of the output. After identifying the large files and directories, you can move them to the appropriate location, (g.g., ``/work`` for research) or you can back up and delete the files and directories if they are no longer required. An example output would look like: ::
+ 
+ [<username>@<host> directory]$  du -shc .[^.]* *
+ 39M	.git
+ 106M	discovery-examples
+ 41K	README.md
+ 3.3M	software-installation
+ 147M	total
+
+Utlilize /scratch and /work
+=======================================================
+For long-term research work storage, ``/work`` should be used. If your PI does not have space in ``/work`` setup, they can request ``/work`` using the `New Storage Space Request <https://bit.ly/NURC-NewStorage>`_. PIs can also request additional storage using the `Storage Space Extension Request <https://bit.ly/NURC-StorageExtension>`_. More details of ``/work`` and ``/scratch`` can be found at :ref:`discovery_storage`. 
+
+For temporary job files, you can utilize ``/scratch/<username>``. Please be mindful of the ``/scratch`` purge policy, which can be found on the `Research Computing Policy Page <https://rc.northeastern.edu/policy/>`_.
+
+Clean ~/.conda directory
+=======================================================
+.. note::
+  Conda environments are part of your research and should preferably be stored in your PI's ``/work`` directory. 
+
+If conda environments are stored in your ``/home/<username>`` directory, here are some suggestions to reduce storage size of the environments.
+
+To remove any unused packages and caches from using conda, from an srun session, ``srun --pty /bin/bash``, load the version of anaconda/minicoda the environment are built with and run: ::
+
+ source activate <your environment>
+ conda clean --all
+
+This will only delete unused packages in your ``~/.conda/pkgs`` directory.
+
+To remove any unused conda environments, run: ::
+
+ conda env list
+ conda env remove --name <your environment>
+
+Clean ~/.singularity directory
+=======================================================
+If you have pulled any containers to Discovery using Singularity, you can clean your container cache in your ``/home/<username>`` directory by running the following command from an srun session (``srun --pty /bin/bash``): ::
+
+ module load singularity/3.5.3
+ singularity cache clean all
+
+To avoid your ``~/.singularity`` directory filling up, you can set a temporary directory for when you pull containers to store the cache in that location; an example of this procedure, (where ``<project>`` is your PI's ``/work`` directory) is: ::
```

Comment:
I can see your adjustment here but I don't know if it would be clear and I don't want people trying to set the temporary directory to /work which with the change, sounds like it might happen. We can talk about this more because there might be a better way to phrase this.